index.rst

```diff
@@ -6,25 +6,251 @@
+        HD 224726,00 00 11.6217,-00 21 37.608,14,14,0.021,0.017,90,A,O,2020yCat.1350....0G,V,7.27,--,Vega,,E,V,1
+        * 5 Cet,00 08 12.0955,-02 26 51.740,14,14,0.059,0.036,90,A,O,2020yCat.1350....0G,V,6.22,--,Vega,,E,V,1
+        HD 406,00 08 43.1091,-02 13 21.296,14,14,0.036,0.030,90,A,O,2020yCat.1350....0G,V,7.05,0.010,Vega,2000A&A...355L..27H,D,V,1
+        HD 1567,00 19 50.8746,-02 28 43.990,14,14,0.028,0.015,90,A,O,2020yCat.1350....0G,V,6.96,0.010,Vega,2000A&A...355L..27H,D,V,1
+        ...,...,...,...,...,...,...,..,...,...,...,...,...,...,,...,...,...
+        HD 1421,00 18 18.5194,-02 00 53.291,14,14,0.021,0.014,90,A,O,2020yCat.1350....0G,V,7.18,--,Vega,,E,V,1
+        HD 999,00 14 24.4641,-02 11 52.802,14,14,0.022,0.018,90,A,O,2020yCat.1350....0G,V,7.18,0.010,Vega,2000A&A...355L..27H,D,V,1
+        HD 820,00 12 40.3372,-01 13 37.885,14,14,0.019,0.015,90,A,O,2020yCat.1350....0G,V,7.2,0.010,Vega,2000A&A...355L..27H,D,V,1
+        HD 1369,00 17 48.3722,-01 51 45.858,14,14,0.068,0.055,90,A,O,2020yCat.1350....0G,V,7.1,0.010,Vega,2000A&A...355L..27H,D,V,1
+        HD 2023,00 24 29.6495,-02 13 08.626,14,14,0.027,0.018,90,A,O,2020yCat.1350....0G,V,6.067,0.010,Vega,2000A&A...355L..27H,D,V,1
+
+
+
+Exclude individual stars manually (optional)
+--------------------------------------------
+This section is to exclude the stars from the list manually. Put HD of the stars on the ``Remove_main_id`` parameter. Even if there are any stars not included in the final table, it is fine to run. 
+
+.. code-block:: py
+    
+    Remove_main_id = ["HD22746","HD452"] #HD NNNNNN 
+    p= re.compile("\d*\.?\d+")
+    customSimbad = Simbad()
+    for i in range(len(Remove_main_id)):
+        number = p.findall(Remove_main_id[i])[0]
+        Remove_main_id_simbad= customSimbad.query_region('HD '+str(number))["MAIN_ID"]
+        Remove = (final["MAIN_ID"] == Remove_main_id_simbad[0])
+        if np.count_nonzero(Remove) !=0 :
+            remove_index = [i for i, x in enumerate(Remove) if x]
+            final.remove_row(remove_index[0])
+            print(Remove_main_id_simbad[0]+' is now removed from the final catalog')
+
+
+Save the catalog on the output file
+-----------------------------------
+As a final step, the queried table is saved into json file. The default name for output is :file:`HD_cwfs_stars.pd`. The file name can changed with ``file_name_final`` variable. 
+
+.. code-block:: py
+   
+    file_name_final = 'HD_cwfs_stars.pd' #file_name
+    result = final.to_pandas().to_json(file_name_final)
+    print('List of Stars was exported to '+file_name_final)
+
+Plot for the distribution of the Stars on Sky
+=============================================
+To check wheather selected CWFS stars are homogeneously distributed on the southern sphere, equatorial coordinates RA, Dec of each starsare plotted on Mollweide projection.
```

Comment:
Two typos; *whether and starsare and it should say "plotted on a Mollweide projection."